Project-Starlord/Controllers/FollowerController.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Newtonsoft.Json;
+using Project_Starlord.Data;
+using Project_Starlord.Models;
+
+namespace Project_Starlord.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class FollowerController : ControllerBase
+    {
+        private readonly MyDbContext _context;
+
+        public FollowerController(MyDbContext context)
+        {
+            _context = context;
+        }
+
+        [Authorize]
+        [HttpPost]
+        public async Task<ActionResult<FollowerModel>> PostFollowerModel(FollowerModel follower)
+        {
+            _context.Followers.Add(follower);
+            await _context.SaveChangesAsync();
+
+            return follower;
+        }
+
+        [Authorize]
+        [HttpDelete("{id}")]
+        public async Task<ActionResult<FollowerModel>> DeleteFollowerModel(int id)
+        {
+            var followerModel = await _context.Followers.FindAsync(id);
+            if (followerModel == null)
+            {
+                return NotFound();
+            }
+
+            _context.Followers.Remove(followerModel);
+            await _context.SaveChangesAsync();
+
+            return followerModel;
+        }
```

Comment:
So you can basically delete any follower if you have the id? Can't that cause any problems?

Reply:
For now it will not be a problem. If it would be bigger then yes.